src/poc/miner_onion_server.erl

```diff
@@ -168,21 +169,28 @@ send_witness(Data, OnionCompactKey, Time, RSSI, Retry) ->
 %% gen_server Function Definitions
 %% ------------------------------------------------------------------
 init(Args) ->
+    lager:info("init with ~p", [Args]),
```

Comment:
Does this add much value?

Reply:
this line was already present, i just moved it up....personally i think its good to have all core servers say hello on startup.  Gives you the warm reassurance they are up and allows you to easily identify the PID when you need it...

Reply:
Fair.